_posts/plater-blog-post.md

```diff
@@ -0,0 +1,104 @@
+---
+name: plater-development
+layout: post
+title: From a million nested `ifelse`s to the plater package
+date: 2017-01-30
+authors:
+  - name: Sean Hughes
+    url: https://github.com/seaaan
+categories:
+  - blog
+tags:
+- ropensci
+- packages
+- plater
+- R
+---
+
+As a lab scientist, I do almost all of my experiments in [microtiter plates](https://en.wikipedia.org/wiki/Microtiter_plate). These tools are an efficient means of organizing many parallel experimental conditions. It's not always easy, however, to translate between the physical plate and a useful data structure for analysis. My first attempts to solve this problem--nesting one `ifelse` call inside of the next to describe which well was which--were very unsatisfying. Over time, my attempts at solving the problem grew more sophisticated, and eventually, the `plater` package was born. Here I will tell the story of how with the help of [R Packages](http://r-pkgs.had.co.nz/) and the [amazing](http://deanattali.com/) [reviewers](http://www.juliagustavsen.com/) and [editors](https://scottchamberlain.info/) at rOpenSci, I ended up with a package that makes it easy to work with plate-based data. 
```

Comment:
Do you think you could name reviewers in the post so that people see their names?  e.g., `amazing reviewers ([Julia Gustavsen](http://www.juliagustavsen.com/) and [Dean Attali](http://deanattali.com/))` instead of `amazing reviewers` with their names hidden

same for below instance of the same

p.s. leave the `editor` bit as is, no need to mention us by name since it's our job :)